diff --git a/GDIbuilder/GDIbuilder.csproj b/GDIbuilder/GDIbuilder.csproj
index 4d4a893..96ee974 100644
--- a/GDIbuilder/GDIbuilder.csproj
+++ b/GDIbuilder/GDIbuilder.csproj
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <UseWindowsForms>true</UseWindowsForms>
     <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
     <OutputType>WinExe</OutputType>
diff --git a/buildgdi/buildgdi.csproj b/buildgdi/buildgdi.csproj
index ea78659..56b13b3 100644
--- a/buildgdi/buildgdi.csproj
+++ b/buildgdi/buildgdi.csproj
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
     <OutputType>Exe</OutputType>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
@@ -19,7 +19,7 @@
     <ProjectReference Include="..\GDIbuilder\DiscUtils\DiscUtils.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-*" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="*" />
     <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
   </ItemGroup>
   <ItemGroup>
